# msg/UltrasonicDistances.msg

float32 front_left
float32 front_right
float32 left
float32 right
float32 rear
